### ISSUE TITLE
How can I print a tensor inside my custom loss function?

### ISSUE THREAD

**Issue by r3db** [tflearn/tflearn#856]
I have this code:
`network = regression(network, optimizer='adam', learning_rate=0.001, loss=my_loss)`

This is how I define my_loss:
```python
def my_loss(y_pred, y_true):
    with tf.name_scope(None):
        return tf.nn.l2_loss(y_pred - y_true)
```

I want to print `y_true`

I tried:
```python
s = tf.Session()
print(s.run(y_true))
```

and

```python
s = tf.Session()
x = tf.Print(y_true, [y_true])
print(x.eval(session = s))
```

I always get the same error message:
![image](https://user-images.githubusercontent.com/9978724/28757193-6b4eda88-7575-11e7-9e6b-893a09f2b806.png)

Can anyone help me?
Cheers

**Comment by ashrafabdul** [comment]
@r3db Were you able to solve this? I'm stuck with the same problem.

**Comment by r3db** [comment]
Unfortunately I was not able to solve my problem
